sycl/test-e2e/ESIMD/unified_memory_api/Inputs/atomic_update_slm.hpp

```diff
@@ -591,6 +595,12 @@ bool test_fp_types(queue q) {
     }
   }
   passed &= run_test<UseAcc, float, N, Op, UseMask>(q);
+  if (q.get_device().has(sycl::aspect::atomic64) &&
+      q.get_device().has(sycl::aspect::fp64)) {
+    // Disable double data type for fcmpxchg operation as D64 data is not
```

Comment:
Why this comment is needed? If fcmpxchg really does not support 'double', then need to disable that case/test, but it seems it is still tested (and if tested, then does it pass?), or I missed something?
If fcmpxchg+'double' supported, then this comment is not needed.

Reply:
It is not supported, the assert requires `sizeof(T) == 2 || sizeof(T) == 4`, removed it.